Training25/Program.cs

```diff
@@ -3,11 +3,44 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T11 - Armstrong number.
 // ------------------------------------------------------------------------------------------------
 namespace Training25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      string input = GetInput ();
+      ArmStrong (input);
    }
-}
+   /// <summary> Gets only postive integer from the user </summary>

```

Comment:
Don't leave a space before or after `<summary>`
```suggestion
   /// <summary>Gets only postive integer from the user</summary></summary>
```